### PR TITLE
test(frontend): scope-correct PreMarketMovers tests (not in spec #250)

### LINKED ISSUE #314: test(frontend): scope-correct PreMarketMovers tests (not in spec #250)
Scope spillover from #250. PreMarketMovers.test.tsx added OOS during coverage ratchet — not in spec or priority list. Retained because excision breaks coverage thresholds. Automatically triaged by MarketHawk Dark Factory scope enforcement.

### PR DESCRIPTION
**Pull request by omniscient**
Closes #314

## Summary
Automated implementation for issue #314.

## Preview
_No preview environment — this change does not affect the running app ('Changeset contains only documentation (docs/codeindex-hotspots.md) and test file (frontend/src/pages/PreMarketMovers.test.tsx). Both are non-runtime-affecting. No backend, frontend source, or infrastructure changes.')._

## Commands
```bash
# Iterate after feedback
docker compose --profile factory run --rm dark-factory "Continue issue #314"

# Tear down preview when done
docker compose --profile factory run --rm dark-factory "Close issue #314"
```

---
*Generated by MarketHawk Dark Factory*

### CODE REVIEW

File: frontend/src/pages/PreMarketMovers.test.tsx [hunk 1]
```diff
@@ -22,4 +42,40 @@ describe('PreMarketMovers page', () => {
     renderWithQuery(<PreMarketMovers />);
```
Comment (omniscient) [inline frontend/src/pages/PreMarketMovers.test.tsx:42]: **maintainability** · **low** — `renderWithQuery` provides a QueryClient but `PreMarketMovers` uses `useState`/`useEffect`, not React Query; the wrapper is harmless overhead here. Not blocking, but a plain `render` would more accurately reflect the component's data flow.

File: frontend/src/pages/PreMarketMovers.test.tsx [hunk 2]
```diff
@@ -22,4 +42,40 @@ describe('PreMarketMovers page', () => {
...
+    renderWithQuery(<PreMarketMovers />);
+    await waitFor(() => {
+      expect(screen.getByText(/No movers found/i)).toBeInTheDocument();
+    });
+  });
+
+  it('filters table rows by ticker text input', async () => {
+    mockFetchPreMarketMovers.mockResolvedValueOnce({
+      status: 'ok',
+      timestamp: '2026-01-01T04:00:00.000Z',
```
Comment (omniscient) [inline frontend/src/pages/PreMarketMovers.test.tsx:65]: **maintainability** · **low** — The filter test uses `mockResolvedValueOnce`; the component schedules `setInterval(fetchData, 60000)`, so any future use of fake timers or a longer-running test would silently fall back to `EMPTY_RESPONSE`. Use `mockResolvedValue` for the duration of the test to make the row data stable regardless of refetch.

File: frontend/src/pages/PreMarketMovers.test.tsx [hunk 3]
```diff
@@ -22,4 +42,40 @@ describe('PreMarketMovers page', () => {
...
+      ],
+    });
+    renderWithQuery(<PreMarketMovers />);
+    await waitFor(() => {
+      expect(screen.getByText('Apple')).toBeInTheDocument();
+    });
+
+    fireEvent.change(screen.getByRole('textbox'), { target: { value: 'AAPL' } });
+
+    expect(screen.queryByText('Nvidia')).not.toBeInTheDocument();
```
Comment (omniscient) [inline frontend/src/pages/PreMarketMovers.test.tsx:78]: **edge-case** · **low** — The filter test depends on the component's default `minVolume` (50000) being below the factory `volume: 100000`; this coupling is invisible. Either keep volume explicitly well above the default with a comment, or it will break if the default minVolume changes — consider asserting both rows render before filtering to catch volume-filter regressions.

**Review by omniscient — COMMENTED** [review]
## 🏭 Dark Factory Code Review

**0** blocking · **3** advisory finding(s).
